tests/cloud_health_test.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+import unittest
+import sys
+
+#
+# Third party libraries
+#
+
+from mock import MagicMock, patch
+
+#
+# Internal libraries
+#
+
+from krux.cli import get_parser, get_group
+from krux.stats import DummyStatsClient
+from krux_cloud_health.cloud_health import CloudHealth, get_cloud_health, Interval, NAME
+
+
+class CloudHealthTest(unittest.TestCase):
+
+    API_KEY = '12345'
+    COST_HISTORY_REPORT = 'olap_reports/cost/history'
+    COST_CURRENT_REPORT = 'olap_reports/cost/current'
+    
+    @patch('krux_cloud_health.cloud_health.get_stats')
+    @patch('krux_cloud_health.cloud_health.get_logger')
+    def setUp(self, mock_logger, mock_stats):
+        self.mock_logger = mock_logger
+        self.mock_stats = mock_stats
+        self.cloud_health = CloudHealth(CloudHealthTest.API_KEY, self.mock_logger, self.mock_stats)
+
+    @patch('krux_cloud_health.cloud_health.get_stats')
+    @patch('krux_cloud_health.cloud_health.get_logger')
+    @patch('krux_cloud_health.cloud_health.get_parser')
+    @patch('krux_cloud_health.cloud_health.CloudHealth')
+    def test_get_cloud_health_no_args(self, mock_cloud_health, mock_parser, mock_logger, mock_stats):
```

Comment:
You want to test the case when you pass all the arguments to `get_cloud_health()` and make sure the content of the `if` blocks are not executed.